apps/marketplace/components/Reports/ReportItem.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import includes from 'lodash/includes'
+import { uniqueID } from '../shared/utils/helpers'
+import * as styles from './ReportItem.scss'
+
+const computeClassname = (base, alt, formatted) =>
+  cx(base, {
+    [alt]: formatted
+  })
+
+const imageRoot = '/static/media/'
+
+const ReportItem = props => {
+  const { heading, subitems } = props
+  const mobile = true
+
+  return (
+    <div className="col-sm-12 report-item">
+      <div className={styles.reportItem}>
+        {heading &&
+          <h2 className={`${styles.reportItemHeading} uikit-display-2`}>
+            {heading}
+          </h2>}
+        {subitems.map((subitem, id = uniqueID()) =>
+          <p className={`${styles.reportSubitemContainer} ${subitem.fullWidth ? 'col-sm-12' : 'col-sm-6'}`} key={id}>
```

Comment:
why are you controlling responsiveness with a prop? This should be done with CSS, we had this discussion with Mark, responsiveness has to be done with css!

Reply:
Agreed - this PR was just undeleting that code but I've fixed this and other issues now.